flytekit/types/file/file.py

```diff
@@ -190,6 +221,18 @@ class _SpecificFormatClass(FlyteFile):
             # Get the type engine to see this as kind of a generic
             __origin__ = FlyteFile
 
+            class AttributeHider(FlyteFile):
```

Comment:
Does it need to be `FlyteFile`?
```suggestion
            class AttributeHider(FlyteFile):
```

Reply:
No, it doesn't, I add it because I think this is more readable.